cra-rxjs-styled-components/src/components/repo-page/RenderContextValue.tsx

```diff
@@ -0,0 +1,8 @@
+import { useRepo } from '../../context/RepoContext';
+
+function RenderContextValue() {
```

Comment:
What is this function used for? I am not so sure I understand why we need this

Reply:
I'm sorry... Let me deprecate it